include/TPP/Dialect/Tpp/TppInterface.td

```diff
@@ -51,6 +51,18 @@ def TppOpInterface : OpInterface<"TppOp"> {
         assert($_op.getOutputs().size() == 1);
         return $_op.getOutputs()[0];
       }]
+    >,
+    InterfaceMethod<
+      /*desc=*/[{
+        Return the inputs.
+      }],
+      /*retTy=*/"ValueRange",
+      /*methodName*/"getInputs",
```

Comment:
Is there any reason to have it? It just calls to the default `getInputs` generated by td.

Reply:
If we use the ops using TppOp interface (see `getEffects`) we don't have `getInputs` because it is attached to the specific op like `MatmulOp`.